index.html

```diff
@@ -30,7 +30,147 @@ <h2 class="connect">LET’S CONNECT</h2>
                 </ul>
         </div>
             </div>
+        </section>      
+        <section class="cardworks">
+            <div class="container">
+                <a href="#"> <img src="img/Snapshoot.png" alt="snapshoot"></a>
+                <h1 class="greet">Tonic</h1>
+                <div class="frame2">
+                    <div class="client">CANOPY</div>
+                    <div ><a href="#"> <img src="img/Counter.png" alt="snapshoot"></a></div>
+                    <div class="role">Back End Dev</div>
+                    <div ><a href="#"> <img src="img/Counter.png" alt="snapshoot"></a></div>
+                    <div class="role">2015</div>
+                </div>
+                    <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="tags">
+                   <p class="tag">html</p> 
+                   <p class="tag">css</p>
+                   <p class="tag">javascript</p>
+                </div>
+                <div class="button">
+                    <p class="seeproj">See Project</p>
+                </div>
+            </div>
+        </section>
```

Comment:
- [Optional] Consider adding a background color to the works section and a border to the cards to match the design template.
*Design template*
![Screenshot 2022-08-19 151839](https://user-images.githubusercontent.com/94551484/185639728-9c686c33-0493-4a36-93a3-9102f55ff535.png)
*your design*
![Screenshot 2022-08-19 151944](https://user-images.githubusercontent.com/94551484/185639804-d15b68e5-8d46-48af-9baf-05545c59d5c4.png)

- [Optional] Please in future projects, kindly make your PR description more descriptive. The PR description should be a summary of all the changes implemented in the pull request. You can read more about PR description [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects).

- [Optional} Please in future projects, kindly make your commit messages more descriptive. This will make your work look professional. A good example of a meaningful commit message is `Update work section` a bad example is `Update`. You can check more about commit messages [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects).